services/logging-operator/4.2.5/defaults/cm.yaml

```diff
@@ -112,7 +112,7 @@ data:
         # Explicitly specify the version here. This should be updated when logging-operator is upgraded.
         # Also, update the image in fluent-bit configuration if the image is upgraded here.
         repository: ghcr.io/mesosphere/dkp-container-images/docker.io/fluent/fluent-bit
-        tag: 2.2.3-d2iq.0
+        tag: 2.2.3-d2iq.1
```

Comment:
Would it be possible to switch to 

```
ghcr.io/mesosphere/dkp-container-images/cr.fluentbit.io/fluent/fluent-bit:2.2.3-d2iq.1
```
so that we have only 1 image in the bundle?